src/ps.rs

```diff
@@ -271,26 +272,33 @@ pub fn create_snapshot(
 
     let mut writer = Writer::from_writer(io::stdout());
 
+    const VERSION: &str = env!("CARGO_PKG_VERSION");
+
     for ((user, job_id, command), job_info) in processes_by_job_id {
+        // FIXME this does not print "none" or "unknown"
+        let gpus_comma_separated: String = job_info
+            .gpu_cards
+            .iter()
+            .map(|&num| num.to_string())
+            .collect::<Vec<String>>()
+            .join(",");
+
```

Comment:
I think that it would be easy to add a line of code here so that if gpus_comma_separated is the empty string then it gets the value "none" instead, this is in keeping with the desired syntax and close enough to the eventual semantics that it is better than using the non-standard empty string in the output.

Reply:
Good workaround. I will add.